pkg/kubelet/kubelet_pods.go

```diff
@@ -1268,10 +1277,15 @@ func getPhase(spec *v1.PodSpec, info []v1.ContainerStatus) v1.PodPhase {
 			stopped++
 			if containerStatus.State.Terminated.ExitCode == 0 {
 				succeeded++
+			} else if isContainerRestarting(pod, &containerStatus) {
+				restarting++
 			}
 		case containerStatus.State.Waiting != nil:
 			if containerStatus.LastTerminationState.Terminated != nil {
 				stopped++
+				if isContainerRestarting(pod, &containerStatus) {
+					restarting++
```

Comment:
restarting is normally associated with boolean state.
Maybe rename the variable to make its meaning clearer.